app/helpers/container_helper/textual_summary.rb

```diff
@@ -109,4 +109,23 @@ def textual_se_linux_level
   def textual_run_as_non_root
     {:label => "Run As Non Root", :value => @record.container_definition.run_as_non_root} unless @record.container_definition.run_as_non_root.nil?
   end
+
+  def textual_group_env
+    {
+      :additional_table_class => "table-fixed",
+      :expandable             => true,
+      :labels                 => [_("Name"), _("Type"), _("value")],
+      :values                 => collect_env
+    }
+  end
+
+  def collect_env
+    @record.container_definition.container_env_vars.collect do |var|
+      [
+        var.name,
+        var.value.nil? ? "REFERENCE" : "VALUE",
```

Comment:
> The value of an env var can be a string (that includes other vars) called value OR a reference to other fields called valueFrom[1]
> 
> [1] https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L701

I wonder if it still requires hardcoded strings on miq side or it can be somehow generically extracted from k8s output